executor.py

```diff
@@ -101,13 +101,13 @@ def __init__(
             )
 
     def _add_to_index(
-        self, vectors: "np.ndarray", ids: List, cluster_center: int, iter: int
-    ):
+        self, vectors: "np.ndarray", ids: List):
         if self._rii_index is None or not self._is_trained:
             self.logger.warning('Please train the indexer first before indexing data')
             return
 
-        self._rii_index.add_configure(vecs=vectors, nlist=cluster_center, iter=iter)
+        self._rii_index.add(vecs=vectors)
+        self._needs_reconfigure = True
```

Comment:
Can you also add a test to assert on this?

- index
- assert
- search
- assert

Something like that?

Reply:
okay, I will add it